ansible/roles/ooni-measurements/tasks/main.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: create group
+  group:
+    name: "oomsmweb"
+    state: present
+
+- name: create user
+  user:
+    name: "oomsmweb"
+    group: "oomsmweb"
+    comment: "OONI API user"
+    state: present
+
```

Comment:
No idea why this diff is showing up, but it's already in master. It's likely due to the merge commit confusing github.